conduit/telemetry/telemetry.go

```diff
@@ -0,0 +1,84 @@
+package telemetry
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/opensearch-project/opensearch-go"
+	"github.com/opensearch-project/opensearch-go/opensearchapi"
+
+	"github.com/algorand/conduit/version"
+)
+
+// MakeTelemetryConfig initializes a new TelemetryConfig.
+func MakeTelemetryConfig(telemetryURI, index, username, password string) Config {
+	return Config{
+		Enable:   true,
+		URI:      telemetryURI,
+		GUID:     uuid.New().String(), // Use Google UUID instead of go-algorand utils
```

Comment:
It seems like an uncontroversial decision to me, but the comment got me curious about the pros/cons.